.github/workflows/continous-integration.yml

```diff
@@ -145,10 +145,11 @@ jobs:
       XSS_PROTECTION: True
       CSP_CONNECT_SRC: "*"
       CSP_FONT_SRC: "'self' https://fonts.gstatic.com https://fonts.googleapis.com https://code.cdn.mozilla.net https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/fonts/ data: https://static.fundraiseup.com/common-fonts/"
-      CSP_FRAME_SRC: "'self' https://www.google.com/recaptcha/ https://*.stripe.com https://pay.google.com https://*.paypal.com"
+      CSP_IMG_SRC: "* data: blob: *.fundraiseup.com ucarecdn.com pay.google.com *.paypalobjects.com"
+      CSP_FRAME_SRC: "'self' https://www.google.com/recaptcha/ https://*.stripe.com https://pay.google.com https://*.paypal.com *.fundraiseup.com"      
       CSP_SCRIPT_SRC: "'self' 'unsafe-inline' https://www.google-analytics.com/analytics.js http://*.shpg.org/ https://comments.mozillafoundation.org/ https://airtable.com https://platform.twitter.com https://cdnjs.cloudflare.com/ajax/libs/gsap/3.8.0/gsap.min.js https://cdnjs.cloudflare.com/ajax/libs/gsap/3.8.0/ScrollTrigger.min.js https://*.googletagmanager.com https://*.fundraiseup.com https://mozillafoundation.tfaforms.net https://www.google.com/recaptcha/ https://www.gstatic.com/recaptcha/ 'unsafe-eval' https://*.stripe.com https://m.stripe.network https://*.paypal.com https://*.paypalobjects.com https://pay.google.com"
       CSP_STYLE_SRC: "'self' 'unsafe-inline' https://code.cdn.mozilla.net https://fonts.googleapis.com https://platform.twitter.com https://mozillafoundation.tfaforms.net https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"
-      SECURE_CROSS_ORIGIN_OPENER_POLICY: "'same-origin-allow-popups'"
```

Comment:
I know the special CSP source keywords like `self` or `unsafe-inline` need to be wrapped in quotation marks. I'm not sure about [SECURE_CROSS_ORIGIN_OPENER_POLICY](https://django.readthedocs.io/en/stable/ref/settings.html#secure-cross-origin-opener-policy). @jhonatan-lopes do you know?

Reply:
I think it doesn't because it's a [straight Django setting](https://docs.djangoproject.com/en/4.2/ref/middleware/#cross-origin-opener-policy) instead of being managed through the `django-csp` package